### ISSUE TITLE
Lag fix

### ISSUE THREAD

**Issue by michaeltblount** [MM2-0/Kvaesitso#1020]
To get rid of lag, disable accessible features in your settings that some apps use. That's the only way I get q smooth experience.

**Comment by jordanfaison** [comment]
Yeah I still have a bit of stuttering when scrolling through the app drawer, definitely not sure what causes it.
